server/routes/api/user.ts

```diff
@@ -69,23 +71,32 @@ userRoutes.route("/confirmation/:branch").post(
 );
 function postApplicationBranchHandler(anonymous: boolean): (request: express.Request, response: express.Response) => Promise<void> {
 	return async (request, response) => {
-		let user: IUserMongoose;
+		let user: Model<IUser>;
 		if (anonymous) {
 			let email = request.body["anonymous-registration-email"] as string;
 			let name = request.body["anonymous-registration-name"] as string;
-			if (await User.findOne({email})) {
+			if (await User.findOne({ email })) {
```

Comment:
So walk-up registration will be done completely outside the ground truth flow. If they later created an account that matched this email using ground truth, would the accounts properly sync up?

Reply:
@joel99 No that's not implemented yet (and I don't think it's covered by an existing issue)
@ehsanmasdar I don't remember if that's the case. We should do some investigating but that shouldn't block HackGT 6 initial release because walk-up won't happen until day-of